test/images/resource-consumer/VERSION

```diff
@@ -1 +1 @@
-1.11
+1.13
```

Comment:
The version bump to 1.13 is intentional.
I'd like to get https://github.com/kubernetes/kubernetes/pull/112924 merged first which bumps image version to 1.12

Reply:
looks like that has merged. good!